bn-models/src/models/_base/base-event.ts

```diff
@@ -4,9 +4,17 @@ import { BoatnetDate, Measurement } from '../_common/index';
 import { Point } from 'geojson'; // GeoJSON https://tools.ietf.org/html/rfc7946
 import { Base } from './base';
 
+export enum Format {
+    DD,
+    DMS,
+    DMM
+}
+
```

Comment:
We want to add string defines to these enums, because if you turn one into JSON (for storing into CouchDB) it will just be a (mysterious) index.
Consider:
```ts
enum Format {
  DD,
  DMS,
  DMM
}
const test = {
  type: "asdf",
  format: Format.DMS
}
const jsonified = JSON.stringify(test);
console.log(jsonified);
```
Outputs:
```console
{"type":"asdf","format":1}
```

Instead,
```ts
export enum Format {
    DD = "DD",
    DMS = "DMS",
    DMM = "DMM"
}
```
Which outputs:
```console
{"type":"asdf","format":"DMS"}
```
In our client code, we will set and compare using those set enum values.